### PR TITLE
fix(intextcitation): Adding initial mark up and styles EHF-13722

### PR DESCRIPTION
**Pull request by yasodaesarapu**

### CODE REVIEW

**Review by wsaun-edh — APPROVED** [review]

**Review by clucaseh — CHANGES_REQUESTED** [review]
Please update your react-component-generator you are not using the latest code

**Comment by yasodaesarapu** [comment]
> Please update your react-component-generator you are not using the latest code

Yes , i will update for my next component creation.

File: src/index.js [hunk 1]
```diff
@@ -1,86 +1,115 @@
...
-          <span className="count">
-            {` ${count} `}
-          </span>
-          times
+      <div className={classNames('in-text-citation__lightbox-container', {
+        'in-text-citation__lightbox-container--mobile': isMobile
+      })}
+      >
+        <div className="in-text-citation__lightbox">
+          <img src={viewSourceIcon} alt="view source icon" className="in-text-citation__lightbox-verifiedsource-icon"/>
```
Comment (clucaseh) [inline src/index.js:106]: Please use img component with resizeOptions

File: src/index.js [hunk 2]
```diff
@@ -1,86 +1,115 @@
...
 
-    const { isActive } = this.state;
+  renderViewSource = () => {
+    const { viewsourceUrl } = this.props;
 
-    return isActive ? activeText : text;
+    return (
+      <Anchor
+        href={viewsourceUrl}
+        className="in-text-citation__lightbox-viewsourceurl"
```
Comment (clucaseh) [inline src/index.js:70]: Why are we not using linkModel with rel passed

File: src/index.js [hunk 3]
```diff
@@ -1,86 +1,115 @@
...
+      >
+        View Source
+        <ArrowDirectional direction="right" strokeWidth="3" transform="rotate(-45)" />
+      </Anchor>
+    );
   }
 
-  render() {
+  renderCloseButton = () => (
+    <button
```
Comment (clucaseh) [inline src/index.js:79]: Why are we not using new button component

File: src/component.scss [hunk 4]
```diff
@@ -1,28 +1,82 @@
...
+    box-shadow: 0 5px 6px 3px color($ui-colors, 'whisper');
+    background-color: color($ui-colors, white);
+    padding: 20px;
+    display: flex;
+    align-items: flex-start;
+    position: relative;
+    justify-content: space-between;
+
+    &-verifiedsource-icon {
+      padding: 12px 15px 0 0;
```
Comment (sblashkin) [inline src/component.scss:18]: I think we should use a spatial system for padding and margins

**Comment by yasodaesarapu** [comment]
@clucaseh @sblashkin Will address your comments in new PR . Right now i am updating component with stories.

**Pull request closed without merging**